### ISSUE TITLE
DTLS negotiation failure

### ISSUE THREAD

**Issue by robdyck** [sipwise/rtpengine#1096]
Using rtpengine very simple UA to call an WEBRTC endpoint. Calling from the simple UA to WEBRTC. The call seems to connect but there is no audio. Rtpengine logs ( level 7 ) show repeated 
```
SRTCP output wanted, but no crypto suite was negotiated
SRTP output wanted, but no crypto suite was negotiated
```
Similar issues have been reported before but the advice given was that the logs would stop after answer. However they repeat as long as the call is up. Wireshark shows a normal handshake up to where the client ( callee ) sends client key exchange. A normal response from the server ( caller ) would be New Session Ticket. There is however no response and Client Key exchange repeats.
Wireshark shows that continuity tests pass.
Pcap available if needed. See attached logs.
[DTLS.txt](https://github.com/sipwise/rtpengine/files/5426997/DTLS.txt)

**Comment by rfuchs** [comment]
Might be the same as #1095. There's a patch linked there that you can try.

**Comment by rfuchs** [comment]
Probably fixed by cd3652e8, now merged

**Comment by robdyck** [comment]
That was it. Thanks
```
Oct 23 08:36:18 slim rtpengine[450310]: INFO: [fhjwnsjcobjfyws@blacky.mylan port 35000]: DTLS: Peer certificate accepted
Oct 23 08:36:18 slim rtpengine[450310]: INFO: [fhjwnsjcobjfyws@blacky.mylan port 35000]: DTLS-SRTP successfully negotiated
Oct 23 08:36:18 slim rtpengine[450310]: INFO: [fhjwnsjcobjfyws@blacky.mylan port 35000]: DTLS-SRTP successfully negotiated
```
